pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -48,12 +48,8 @@ func IsMemoryResourceAvailable(isvc *v1beta1api.InferenceService, totalReqMemory
 
 	container := isvc.Spec.Predictor.GetImplementation().GetContainer(isvc.ObjectMeta, isvc.Spec.Predictor.GetExtensions(), isvcConfig)
 
-	if constants.InferenceServiceContainerName == container.Name {
-		predictorMemoryLimit := container.Resources.Limits.Memory()
-		return predictorMemoryLimit.Cmp(totalReqMemory) >= 0
-	}
-
-	return false
+	predictorMemoryLimit := container.Resources.Limits.Memory()
```

Comment:
Any reason to remove the check here?

Reply:
@yuzisun In ServingRuntime implementation, the container name would be nil since it is set while resolving runtime spec.